libcst/tool.py

```diff
@@ -636,6 +636,38 @@ def _initialize_impl(proc_name: str, command_args: List[str]) -> int:
     return 0
 
 
+def _recursive_find(base_dir: str, base_module: str) -> List[Tuple[str, object]]:
```

Comment:
Can we use `glob.glob` or `Path.rglob` to find all files in a simpler way?

Reply:
Possibly, but we'd still need to mangle the directory names to module names, and we don't get much from glob or rglob.